Resources/doc/index.md

```diff
@@ -124,6 +117,50 @@ In order to add your custom providers implementing `Swap\ProviderInterface`, you
 </service>
 ```
 
+## Cache
+
+Some doctrine cache providers are implemented:
+    - apc
+    - array
+    - xcache
+    - wincache
+    - zenddata
+
+```yaml
+# app/config/config.yml
+florianv_swap:
+    cache:
+        doctrine: apc
+    providers:
+        yahoo_finance: ~
+        google_finance: ~
+```
+
+## Priority
+
+A priority attribute is available in tags and config.yml to sort providers.
+
+```yaml
+# app/config/config.yml
+florianv_swap:
+    providers:
+        yahoo_finance:
+            priority: 10
+        google_finance:
+            priority: 0
+```
+
+```xml
+<service id="acme_demo.provider.custom" class="Acme\DemoBundle\Provider\Custom">
+    <tag name="florianv_swap.provider" priority="5" />
+</service>
+```
+
+In this case, the order of providers will be:
```

Comment:
The order should be (from highest to lowest priority):

- yahoo_finance
- acme_demo.provider.custom
- google_finance